deploy_carvel/local-user-authenticator/README.md

```diff
@@ -0,0 +1,17 @@
+# local-user-authenticator
+
```

Comment:
We can delete these README files, but keep the "how to run this" at the top of the new script.